Makefile

```diff
@@ -23,3 +23,19 @@ VISIBLE_WEB_HOST ?= geomapfish-demo.camptocamp.com
 include CONST_Makefile
 
 secrets.tar.bz2.gpg: amazonses_smtp.txt amazonses_s3.txt
+
+.PHONY: edit-qgis-project
+edit-qgis-project:
+	IP=$$(ip addr show dev $$(route | grep default | awk '{print $$(NF)}' | head -1)  | awk '$$1 ~ /^inet/ { sub("/.*", "", $$2); print $$2 }' | head -1); \
+	docker run -ti \
+	  --add-host=db:$$IP \
+	  --volume $$PWD/qgisserver:/project \
+	  --volume /tmp/.X11-unix:/tmp/.X11-unix \
+	  --volume $$HOME:$$HOME \
+	  --env DISPLAY=unix$${DISPLAY} \
+	  --rm camptocamp/qgis-server:latest /usr/local/bin/start-client.sh
+
+qgisserver/project.qgs.tmpl: qgisserver/project.qgs Makefile
+	sed -e "s/user='[^']*'/user='\$${PGUSER}'/g; s/password='[^']*'/password='\$${PGPASSWORD}'/g; s/host=db/host=\$${PGHOST}/g; s/port=[0-9]*/port=\$${PGPORT}/g; s/dbname='[^']*'/dbname='\$${PGDATABASE}'/g" < $< > $@
```

Comment:
With that we can't have more than one database, right?

Reply:
Exact.

Reply:
And what about doing one repacement to touch only the database host db?
I can do it in an other pull request if needed...

Reply:
Good idea, changing that.